README.md

```diff
@@ -72,7 +71,7 @@ $ tapioca help
 
 Commands:
   tapioca --version, -v      # Show version
-  tapioca annotations        # Pull gem RBI annotations from remote sources
+  tapioca annotations        # Deprecated, this command has been merged into `tapioca gem`. Please use that instead.
```

Comment:
I think we should keep the original description but add `deprecated`.

The error message when we run the command already explains what to run.